core/tests/protractor/collections.js

```diff
@@ -20,14 +20,21 @@ var general = require('../protractor_utils/general.js');
 var users = require('../protractor_utils/users.js');
 var AdminPage = require('../protractor_utils/AdminPage.js');
 var collectionEditor = require('../protractor_utils/collectionEditor.js');
+var CreatorDashboardPage =
+  require('../protractor_utils/CreatorDashboardPage.js');
 
 
 describe('Collections', function() {
   var adminPage = null;
+  var creatorDashboardPage = null;
+  var COLLECTION_ID = null;
 
   beforeAll(function() {
     adminPage = new AdminPage.AdminPage();
+    creatorDashboardPage = new CreatorDashboardPage.CreatorDashboardPage();
     var USERNAME = 'aliceCollections';
+    var USERNAME_TWO = 'playerCollections';
```

Comment:
Call them EDITOR_USERNAME and PLAYER_USERNAME or similar, otherwise the person reading the test will need to keep remembering which is which.

Reply:
Ok, will change that.

Reply:
Done